src/pages/facility_confirmation.js

```diff
@@ -1,48 +1,41 @@
 /** @jsx jsx */
-import { useEffect, useState } from 'react';
 import { jsx } from '@emotion/core';
-import { useHistory, useParams } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
 
 import { Routes } from 'constants/Routes';
 import { routeWithParams } from 'lib/utils/routes';
+import { useStateValue } from 'state/StateProvider';
 
 import Page from 'components/layouts/Page';
 import PageLoader from 'components/Loader/PageLoader';
 import { FacilityConfirmation as FacilityConfirmationComponent } from 'components/Confirmation';
 
-function FacilityConfirmation({ backend }) {
+function FacilityConfirmation() {
   const history = useHistory();
-  const params = useParams();
+  const [{ pendingFacility }] = useStateValue();
```

Comment:
Maybe `pendingFacility` isn't the right name here. I'm open to ideas!

Reply:
I think it makes sense 👍